draft-pauly-add-requirements.md

```diff
@@ -186,68 +186,82 @@ Using a content-provider's encrypted resolver can also provide several privacy a
 and the content provider, assuming that the content provider would already see the names upon a secure connection
 later being made based on the DNS answers (e.g., in the TLS SNI extension)
 
-# Distilled discovery mechanisms {#mechanisms}
+# Discovery mechanisms {#mechanisms}
 
-The various use cases described in {{use-cases}} do not all necessarily require separate mechanisms.
+The use cases described in {{use-cases}} do not all necessarily require separate mechanisms.
 
-Generally, the use cases can be summarized in two catgories:
+Generally, the use cases can be summarized in two categories:
 
-1. Discover encrypted resolvers equivalent to (or associated with) unencrypted resolvers. (Network-provisioned,
-client-selected, VPN)
-2. Discover encrypted resolvers applicability to a limited set of domains. (Private names, local content, CDN content)
+1. Encrypted resolver discovery: Discover encrypted resolvers equivalent to (or associated with) unencrypted resolvers.
```

Comment:
I think "encrypted resolver discovery" is too broad here. That's not the sub-category, that's the overall document! Instead, this one is specifically about resolved equivalent to/associated with known resolvers.

Reply:
Maybe "associated encrypted resolver discovery"? (Associated to and equivalent are somewhat vague as well, so I'm not sure how to sharpen this.)